server.js

```diff
@@ -133,7 +142,7 @@ api.all(/(.*)/, bodyParser.json(), async (req, res) => {
     method: req.method,
     url: req.url,
     body: req.body,
-    processes: processes,
+    processes: result.processes,
```

Comment:
We can easily get `undefined` passed here if we fall into the `catch` clause a few lines before. We may want to return an error response in this `catch` clause and bypass the Elm app entirely when it happens.

Reply:
Perfect 👌